apps/docs/docs/api/javascript/types.mdx

```diff
@@ -0,0 +1,67 @@
+---
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+sidebar_position: 7
+---
+
+# `stylex.types.*`
+
+A set of helper functions to be used within [`stylex.defineVars`](./defineVars.mdx)
+and [`stylex.createTheme`](./createTheme.mdx) to define CSS types for variables.
+
+These functions result in `@property` CSS rules for the variables with the appropriate
+`syntax` value.
+
+### Example use:
+
+The `types.*` functions are compatible with all patterns that are supported
+by `stylex.defineVars` and `stylex.createTheme` already.
+
+For example, consider the following set of variables:
+
+```tsx title="vars.stylex.js"
+import * as stylex from '@stylexjs/stylex';
+
+export const colors = stylex.defineVars({
+  accent: {
+    default: 'blue',
+    '@media (prefers-color-scheme: dark)': 'lightblue',
+  },
+  sm: '4px',
+});
+```
+
+You can give the two variables types like so:
+
+```tsx title="vars.stylex.js"
+import * as stylex from '@stylexjs/stylex';
+
+export const colors = stylex.defineVars({
+  accent: stylex.types.color({
+    default: 'blue',
+    '@media (prefers-color-scheme: dark)': 'lightblue',
+  }),
+  sm: stylex.types.length('4px'),
+});
+```
+
+## Available types
+
+The following types are available:
+
+- `stylex.types.angle`
+- `stylex.types.color`
+- `stylex.types.url`
+- `stylex.types.image`
+- `stylex.types.integer`
+- `stylex.types.lengthPercentage`
```

Comment:
More documentation?

Reply:
There isn't a lot to say, as it just sets the type in CSS. I guess I'll list the appropriate CSS `syntax` types.